lms/templates/learner_dashboard/course_entitlement.underscore

```diff
@@ -0,0 +1,32 @@
+<div id="change-session-<%- entitlementUUID %>" class="message is-shown">
+    <div class="action-header">
+        <% if (currentSessionId) { %>
+            <%- gettext('Change to a different session or leave the current session.')%>
+        <% } else { %>
+            <%- gettext('To access the course, select a session.')%>
+        <% } %>
+    </div>
+    <div class="action-controls">
+        <select class="session-select" aria-label="<%- StringUtils.interpolate( gettext('Session Selection Dropdown for {courseName}'), { courseName: courseName }) %>">
+            <% _.each(availableSessions, function(session) { %>
+                <option data-session_id="<%- session.session_id %>">
+                    <%- session.session_dates %>
```

Comment:
It might be better to interpolate this into the translation strings.

Reply:
Changed